osquery/config/config.cpp

```diff
@@ -141,6 +141,11 @@ inline void additionalScheduledQuery(const std::string& name,
   ScheduledQuery query;
   query.query = node.second.get<std::string>("query", "");
   query.interval = node.second.get<int>("interval", 0);
+  if (query.interval == 0) {
```

Comment:
This is also an oversight, an invalid interval + splay = 0 could lead to an exception when the schedule starts.